docusaurus.config.js

```diff
@@ -31,6 +31,20 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
         },
       }),
     ],
+    [
+      'redocusaurus',
+      ({
+        specs: [
+          {
+            spec: 'https://api.accounts.firefox.com/swagger.json',
+            route: '/api',
```

Comment:
Will other docs be revised to use Swagger? If so, should we update the route to be along the lines of `/api/auth-server`?

Reply:
sure